cmd/controller/deploy_cluster_scope.yaml

```diff
@@ -174,6 +174,10 @@ spec:
         application: skupper-controller
```

Comment:
After creating a `skupper` namespace and deploy the controller in it with  `deploy_cluster_scope.yaml` the controller shows the following error:
```
» kubectl logs skupper-controller-786c86774d-8wq8b                       
flag provided but not defined: -enable-grants
Usage of /app/controller:
  -version
    	Report the version of the Skupper Site Controller
```

Reply:
Is it because `quay.io/skupper/controller:v2-latest` does not include these changes?

Reply:
Yes, it will be. You can use quay.io/gordons/controller:v2-latest or build your own. I didn't want to push a new image until this was merged.